app/common/templates/common/macros/status.html

```diff
@@ -3,9 +3,10 @@
 {% macro status(status) %}
   {%
     set classMap = {
-        enum.submission_status.NOT_STARTED: "govuk-tag--grey",
-        enum.submission_status.IN_PROGRESS: "govuk-tag--blue",
-        enum.submission_status.COMPLETED: "govuk-tag--green",
+        enum.tasklist_task_status.NOT_STARTED: "govuk-tag--grey",
+        enum.tasklist_task_status.IN_PROGRESS: "govuk-tag--blue",
+        enum.tasklist_task_status.COMPLETED: "govuk-tag--green",
+        enum.tasklist_task_status.NO_QUESTIONS: "govuk-tag--red",
```

Comment:
Is there a file missing that would have this actually calculated and this enum returned instead of a submission status enum? I might just be missing something big!

Reply:
i've added a get_tasklist_status_for_form to the submission helper